libs/langchain-community/src/embeddings/deepinfra.ts

```diff
@@ -0,0 +1,210 @@
+import axios, {AxiosInstance, AxiosResponse} from "axios";
+
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import { Embeddings, EmbeddingsParams } from "@langchain/core/embeddings";
+import { chunkArray } from "@langchain/core/utils/chunk_array";
+
+/**
+ * The default model name to use for generating embeddings.
+ */
+const DEFAULT_MODEL_NAME = "sentence-transformers/clip-ViT-B-32";
```

Comment:
This default is different than the default specified in the example above

Reply:
Thanks for noticing. I missed that part.